### PR TITLE
Added Feature: View To-Do List

### PR DESCRIPTION
**Pull request by naufal-abdillah**
I've added an endpoint for customers to view their to-do tasks list. This feature includes pagination. The Customer only looks at their data and the data hasn't been deleted. 

File(s) modified:

- app/Http/Controllers/**TaskController.php**
- routes/**api.php**

Feature added:

- View To-do: Customers can view their to-do tasks list, by doing a GET request via endpoint `/api/v1/todo/list?page=1` This feature includes pagination. To access a certain page of the list, the customer use `page` as a parameter and the page number as the value. The Customer can only look at their data and the data hasn't been deleted. Corresponding function/method that handles this feature: `TaskController::view()`

### CODE REVIEW

File: app/Http/Controllers/TaskController.php [hunk 1]
```diff
@@ -43,4 +43,15 @@ public function create(Request $request)
...
     }
+    public function view()
+    {
+        // geting the user id from token
+        $currentUser = JWTAuth::user()->id;
+        // Selecting the customer's tasks'id and title] where 'is_deleted' is false
+        $data = Tasks::where([['customer_id', $currentUser],['is_deleted', 0]])->select('id', 'title')->paginate()->items();
+        // Response Message
+        $status = "success";
+        $message = "Data berhasil disimpan";
```
Comment (cakazies) [inline app/Http/Controllers/TaskController.php]: ini kenapa ada data berhasil di simpan ??

File: app/Http/Controllers/TaskController.php [hunk 2]
```diff
@@ -43,4 +43,15 @@ public function create(Request $request)
         $message = "Data berhasil disimpan";
         return response()->json(compact('status', 'message'), 201);
     }
+    public function view()
+    {
+        // geting the user id from token
+        $currentUser = JWTAuth::user()->id;
```
Comment (cakazies) [inline app/Http/Controllers/TaskController.php:49]: di validais dulu kalan dia 0 bagaimana ?

File: app/Http/Controllers/TaskController.php [hunk 3]
```diff
@@ -43,4 +43,15 @@ public function create(Request $request)
...
+        // Selecting the customer's tasks'id and title] where 'is_deleted' is false
+        $data = Tasks::where([['customer_id', $currentUser],['is_deleted', 0]])->select('id', 'title')->paginate()->items();
+        // Response Message
+        $status = "success";
+        $message = "Data berhasil disimpan";
+        return response()->json(compact('status', 'message','data'), 200);
```
Comment (cakazies) [inline app/Http/Controllers/TaskController.php:55]: ini uda di coba kah ? harusnya ga muncul dataya ini

File: app/Http/Controllers/TaskController.php (hunk 1 above)
Comment (naufal-abdillah) [reply app/Http/Controllers/TaskController.php]: oiya mas, salah message, udah bikin commit baru ganti messagenya

File: app/Http/Controllers/TaskController.php (hunk 2 above)
Comment (naufal-abdillah) [reply app/Http/Controllers/TaskController.php:49]: __Oke mass, kutambah validasi abis nyimpen  JWTAuth::user()

**Review by cakazies — APPROVED** [review]

**Pull request merged**